tests/fixtures/noqa.py

```diff
@@ -345,3 +345,11 @@ def multiple_return_path():
         return 2
     else:
         return 3
+
+
+def bad_default_values(
+    bad,
+    defaults=(),
```

Comment:
These values are not bad. We allow strings, tuples, and other primitives. 
You need to provide wrong values here.

Reply:
Oh, my bad! Thank you!